ext/riverlea/settings/riverlea.setting.php

```diff
@@ -47,4 +47,30 @@
       'display' => ['section' => 'theme', 'weight' => 40],
     ],
   ],
+  'riverlea_font_size' => [
+    'name' => 'riverlea_font_size',
+    'group' => 'riverlea',
+    'default' => '1',
+    'type' => 'String',
+    'html_type' => 'select',
+    'add' => 1.0,
+    'options' => [
+      '0.75' => E::ts('Smallest'),
+      '0.875' => E::ts('Small'),
+      '1' => E::ts('Default'),
+      '1.125' => E::ts('Big'),
+      '1.5' => E::ts('Bigger'),
+    ],
+    'on_change' => [
+      '\\Civi\\Riverlea\\StyleLoader::onChangeFontsize',
+    ],
+    'validate_callback' => '\\Civi\\Riverlea\\StyleLoader::validateFontsize',
+    'title' => E::ts('Font size'),
+    'is_domain' => 1,
+    'is_contact' => 0,
+    'description' => E::ts('The font size (Smallest 12px, Small 14px, Default 16px, Big 18px, Bigger 24px)'),
```

Comment:
These describe in pixels. But the options are actually `rem` values. Should that be made consistent?